### PR TITLE
contributing

### PR DESCRIPTION
**Pull request by Varakala02**

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -406,3 +406,10 @@ Example:
 i.MX 8MP uses a different ``mmc`` command from master and from the rest of the
 i.MX 8M family. The content will be split again and i.MX 8MP needs to use its
 own specific version (different file).
+
+How to use substitution after dot "." in descriptions?
+------------------------------------------------------
+
+To  use substitution variable in descriptions after a dot "." it need to 
+specified as the following "./ |substitution_variable|" in descriptions. A sample 
+example is below.
```
Comment (jonas-rem) [inline CONTRIBUTING.rst:415]: You were describing an example, but as of now there is no example.

**Review by jonas-rem — CHANGES_REQUESTED** [review]
@Varakala02 do you still work on this? If not, I would close it for now.

File: CONTRIBUTING.rst (hunk 1 above)
Comment (Varakala02) [reply CONTRIBUTING.rst:415]: Hi @jonas-rem Sorry, I totally forgot this, thank you. I will update it.

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -465,3 +465,16 @@ Example:
 i.MX 8MP uses a different ``mmc`` command from master and from the rest of the
 i.MX 8M family. The content will be split again and i.MX 8MP needs to use its
 own specific version (different file).
+
+How to use substitution after full stop (.) in descriptions?
+------------------------------------------------------
```
Comment (jonas-rem) [inline CONTRIBUTING.rst:470]: The underline length should match the text length.

Otherwise looks good.

**Review by jonas-rem — CHANGES_REQUESTED** [review]

**Review by jonas-rem — APPROVED** [review]

**Pull request merged**